From: Feignint <37488595+FeignInt@users.noreply.github.com>
Date: Sat, 28 Nov 2020 08:01:06 +0000
Subject: Set soundfont to default-GM.sf2 in midi_fluidsynth.cpp
Forwarded: not-needed

On Debian libfluidsynth2 depends on:

    timgm6mb-soundfont | sf3-soundfont-gm

Either will prime the Debian Alternatives system
with default-GM.sf2

This patch changes the default soundfont from
default.sf2 to default-GM.sf2
---
 src/midi/midi_fluidsynth.cpp | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/midi/midi_fluidsynth.cpp b/src/midi/midi_fluidsynth.cpp
index e9aacdb..9e57b2f 100644
--- a/src/midi/midi_fluidsynth.cpp
+++ b/src/midi/midi_fluidsynth.cpp
@@ -38,9 +38,8 @@ static void init_fluid_dosbox_settings(Section_prop &secprop)
 {
 	constexpr auto when_idle = Property::Changeable::WhenIdle;
 
-	// Name 'default.sf2' picks the default soundfont if it's installed
-	// it the OS. Usually it's Fluid_R3.
-	auto *str_prop = secprop.Add_string("soundfont", when_idle, "default.sf2");
+	// Name 'default-GM.sf2' Debian alternatives system for sf2-soundfont-gm
+	auto *str_prop = secprop.Add_string("soundfont", when_idle, "default-GM.sf2");
 	str_prop->Set_help(
 	        "Path to a SoundFont file in .sf2 format. You can use an\n"
 	        "absolute or relative path, or the name of an .sf2 inside\n"
